packages/core/test-helpers/core-test-helpers-model-versions/README.md

```diff
@@ -0,0 +1,19 @@
+# @kbn/core-test-helpers-model-versions
+
+Package exposing utilities for model version integration testing.
+
+This package exposes a `createModelVersionTestBed` utility which allow simulating
+a testbed environment where we're in the cohabitation period between two versions, to test the interactions
+between two model versions of a set of SO types.
```

Comment:
Given the next stage of the plan is to have proper testing documentation, I don't think we need to go into more details in the readme?